platform/conf/conf.go

```diff
@@ -224,6 +224,40 @@ func (c *Conf) Bytes() []byte {
 	return []byte(c.String())
 }
 
+func (c *Conf) addSystemdUnitV1(name, contents string, enable bool) {
+	c.ignitionV1.Systemd.Units = append(c.ignitionV1.Systemd.Units, v1types.SystemdUnit{
+		Name: v1types.SystemdUnitName(name),
+		Contents: contents,
+		Enable: enable,
+	})
+}
+
+func (c *Conf) addSystemdUnitV2(name, contents string, enable bool) {
+	c.ignitionV2.Systemd.Units = append(c.ignitionV2.Systemd.Units, v2types.Unit{
+		Name: name,
+		Contents: contents,
+		Enable: enable,
```

Comment:
Oh, yes, you're right.  Go ahead and leave it as `Enable` for now.  Once https://github.com/coreos/ignition/pull/418 merges, I'll be changing `platform/conf` to use separate data structures for 2.0 and 2.1 configs, and we can fix it then.